api/Storage.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "Storage": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Storage",
+        "support": {
+          "webview_android": {
+            "version_added": "2.1"
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "2"
+          },
+          "firefox_android": {
+            "version_added": null
```

Comment:
Firefox android is the same as Firefox desktop, except where otherwise noted or where Firefox desktop < 4, in which case Firefox android is 4.